ERROR - 2024-01-24 08:42:35 --> Error connecting to the database: mysqli_sql_exception: Uma tentativa de conexão falhou porque o componente conectado não respondeu
corretamente após um período de tempo ou a conexão estabelecida falhou
porque o host conectado não respondeu in C:\laragon\www\CodeIgniter\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\laragon\www\CodeIgniter\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('178.128.244.159', 'higorzica', 'Olavo34763455@', 'durabi91_portfo...', 3306, '', 0)
#1 C:\laragon\www\CodeIgniter\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\CodeIgniter\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\CodeIgniter\app\Controllers\Home.php(12): CodeIgniter\Database\BaseConnection->query('SELECT proj.id,...')
#4 C:\laragon\www\CodeIgniter\system\CodeIgniter.php(934): App\Controllers\Home->index()
#5 C:\laragon\www\CodeIgniter\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\laragon\www\CodeIgniter\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\laragon\www\CodeIgniter\public\index.php(67): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Uma tentativa de conexão falhou porque o componente conectado não respondeu
corretamente após um período de tempo ou a conexão estabelecida falhou
porque o host conectado não respondeu in C:\laragon\www\CodeIgniter\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\laragon\www\CodeIgniter\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\CodeIgniter\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\CodeIgniter\app\Controllers\Home.php(12): CodeIgniter\Database\BaseConnection->query('SELECT proj.id,...')
#3 C:\laragon\www\CodeIgniter\system\CodeIgniter.php(934): App\Controllers\Home->index()
#4 C:\laragon\www\CodeIgniter\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 C:\laragon\www\CodeIgniter\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\laragon\www\CodeIgniter\public\index.php(67): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-01-24 08:42:35 --> Unable to connect to the database.
Main connection [MySQLi]: Uma tentativa de conexão falhou porque o componente conectado não respondeu
corretamente após um período de tempo ou a conexão estabelecida falhou
porque o host conectado não respondeu
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\Home.php(12): CodeIgniter\Database\BaseConnection->query('SELECT proj.id, proj.titulo, proj.descricao, proj.link_github, proj.link_demo, proj.foto
        FROM projetos AS proj WHERE ativo = \'ativo\'')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-24 08:44:59 --> Error connecting to the database: mysqli_sql_exception: Uma tentativa de conexão falhou porque o componente conectado não respondeu
corretamente após um período de tempo ou a conexão estabelecida falhou
porque o host conectado não respondeu in C:\laragon\www\CodeIgniter\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\laragon\www\CodeIgniter\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('178.128.244.159', 'higorzica', 'Olavo34763455@', 'durabi91_portfo...', 3306, '', 0)
#1 C:\laragon\www\CodeIgniter\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\CodeIgniter\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\CodeIgniter\app\Controllers\Home.php(12): CodeIgniter\Database\BaseConnection->query('SELECT proj.id,...')
#4 C:\laragon\www\CodeIgniter\system\CodeIgniter.php(934): App\Controllers\Home->index()
#5 C:\laragon\www\CodeIgniter\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\laragon\www\CodeIgniter\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\laragon\www\CodeIgniter\public\index.php(67): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Uma tentativa de conexão falhou porque o componente conectado não respondeu
corretamente após um período de tempo ou a conexão estabelecida falhou
porque o host conectado não respondeu in C:\laragon\www\CodeIgniter\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\laragon\www\CodeIgniter\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\CodeIgniter\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\CodeIgniter\app\Controllers\Home.php(12): CodeIgniter\Database\BaseConnection->query('SELECT proj.id,...')
#3 C:\laragon\www\CodeIgniter\system\CodeIgniter.php(934): App\Controllers\Home->index()
#4 C:\laragon\www\CodeIgniter\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 C:\laragon\www\CodeIgniter\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\laragon\www\CodeIgniter\public\index.php(67): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-01-24 08:44:59 --> Unable to connect to the database.
Main connection [MySQLi]: Uma tentativa de conexão falhou porque o componente conectado não respondeu
corretamente após um período de tempo ou a conexão estabelecida falhou
porque o host conectado não respondeu
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\Home.php(12): CodeIgniter\Database\BaseConnection->query('SELECT proj.id, proj.titulo, proj.descricao, proj.link_github, proj.link_demo, proj.foto
        FROM projetos AS proj WHERE ativo = \'ativo\'')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
INFO - 2024-01-24 08:48:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-24 09:14:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-24 09:14:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-24 09:14:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-24 09:14:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-24 09:15:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-24 09:15:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-01-24 09:15:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-01-24 12:04:12 --> Error connecting to the database: mysqli_sql_exception: Uma tentativa de conexão falhou porque o componente conectado não respondeu
corretamente após um período de tempo ou a conexão estabelecida falhou
porque o host conectado não respondeu in C:\laragon\www\CodeIgniter\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\laragon\www\CodeIgniter\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('178.128.244.159', 'higorzica', 'Olavo34763455@', 'durabi91_portfo...', 3306, '', 0)
#1 C:\laragon\www\CodeIgniter\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\CodeIgniter\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\CodeIgniter\app\Controllers\Home.php(12): CodeIgniter\Database\BaseConnection->query('SELECT proj.id,...')
#4 C:\laragon\www\CodeIgniter\system\CodeIgniter.php(934): App\Controllers\Home->index()
#5 C:\laragon\www\CodeIgniter\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\laragon\www\CodeIgniter\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\laragon\www\CodeIgniter\public\index.php(67): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Uma tentativa de conexão falhou porque o componente conectado não respondeu
corretamente após um período de tempo ou a conexão estabelecida falhou
porque o host conectado não respondeu in C:\laragon\www\CodeIgniter\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\laragon\www\CodeIgniter\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\CodeIgniter\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\CodeIgniter\app\Controllers\Home.php(12): CodeIgniter\Database\BaseConnection->query('SELECT proj.id,...')
#3 C:\laragon\www\CodeIgniter\system\CodeIgniter.php(934): App\Controllers\Home->index()
#4 C:\laragon\www\CodeIgniter\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 C:\laragon\www\CodeIgniter\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\laragon\www\CodeIgniter\public\index.php(67): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-01-24 12:04:12 --> Unable to connect to the database.
Main connection [MySQLi]: Uma tentativa de conexão falhou porque o componente conectado não respondeu
corretamente após um período de tempo ou a conexão estabelecida falhou
porque o host conectado não respondeu
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\Home.php(12): CodeIgniter\Database\BaseConnection->query('SELECT proj.id, proj.titulo, proj.descricao, proj.link_github, proj.link_demo, proj.foto
        FROM projetos AS proj WHERE ativo = \'ativo\'')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
